app/config/config.yml

```diff
@@ -192,6 +198,21 @@ doctrine:
 doctrine_migrations:
     dir_name: "%kernel.root_dir%/DoctrineMigrations"
 
+# Swiftmailer Configuration
+swiftmailer:
+    spool:     { type: memory }
+    transport: "%mailer_transport%"
+    host:      "%mailer_host%"
+    username:  "%mailer_user%"
+    password:  "%mailer_password%"
+    # using an env var throws an error
+    # auth_mode: "%mailer_auth_mode%"
+    # auth_mode: "%env(MAILER_AUTH_MODE)%"
+    auth_mode: cram-md5
+    port:      "%mailer_port%"
+    # to disable mail uncomment disable_delivery: true
+    # disable_delivery: true
```

Comment:
This should be configurable via an environment variable, direct changes to the `config.yml` file is not desired for deployments (only during development).

Reply:
I wasn't able to find a fix for this. We might want to make this an issue to solve later.